app/views_redesigned/async_property_card_footers/show.html.erb

```diff
@@ -45,4 +45,3 @@
     </div>
   <% end %>
 </div>
-
```

Comment:
Can you revert this change? We would like to always have a new line at the end of each file.

Reply:
Oops. Fixed.